lib/ansible/modules/packaging/language/pip.py

```
@@ -456,6 +520,9 @@ def main():
     umask = module.params['umask']
     env = module.params['virtualenv']
 
+    if NO_SETUPTOOLS:
```

move this before var assignments to fail early.